analyzers/its/expected/akka.net/Akka.Remote--netstandard2.0-S2259.json

```diff
@@ -0,0 +1,30 @@
+{
+"issues":  [
+{
+"id":  "S2259",
+"message":  "'deadLetter' is null on at least one execution path.",
+"location":  {
+"uri":  "sources\akka.net\src\core\Akka.Remote\RemoteActorRefProvider.cs",
+"region":  {
+"startLine":  802,
+"startColumn":  58,
+"endLine":  802,
+"endColumn":  68
+}
+}
+},
```

Comment:
False positive. If `deadLetter` is null, the pattern matching will fail.
https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/sources/akka.net/src/core/Akka.Remote/RemoteActorRefProvider.cs#L800